src/components/Calendar.jsx

```diff
@@ -200,16 +208,56 @@ class Calendar extends React.Component {
           </div>
         </StyledCalendarTitle>
         <CalendarHeader values={calendar[0]} />
-        <CalendarRow values={calendar[1]} handleDayClick={this.interceptDayClick} />
-        <CalendarRow values={calendar[2]} handleDayClick={this.interceptDayClick} />
-        <CalendarRow values={calendar[3]} handleDayClick={this.interceptDayClick} />
-        <CalendarRow values={calendar[4]} handleDayClick={this.interceptDayClick} />
+        <CalendarRow
```

Comment:
Just a suggestion, you might be able to cut down some lines of code if you use map function to return CalendarRow!

Reply:
Thanks, excellent point! Will implement this in a future PR.